modules/sdk-coin-algo/src/algo.ts

```diff
@@ -504,23 +513,6 @@ export class Algo extends BaseCoin {
     return true;
   }
 
-  /** @inheritDoc */
-  deriveKeyWithSeed({ key, seed }: { key: string; seed: string }): { derivationPath: string; key: string } {
```

Comment:
Are there any breaking changes associated with the removal of this function in regards to the output of this versus what is now inherited?

Reply:
It is but also is an incomplete feature that never worked.